### DISCUSSION TITLE
No live data

### DISCUSSION THREAD

**Discussion by vressel** [jobvk/Home-Assistant-Windcentrale#148] [General]
Since it's not an software-issue maybe this is the place to start a discussion about the problem of not getting live data from my windmill (De Blauwe Reiger) since 4 okt 10h00.
Anybody else experiencing the same issue? The live-data in the app is also not working.
Allready sent a mail to the helpdesk of de windcentrale, but no response.

**Comment by Xorfor** [comment]
The same here. Since 4-okt 12:00 no data for De Trouwe Wachter and De Grote Geert. Yesterday I send them an e-mail (again, I think for the 6th time this year).

**Comment by jobvk** [comment]
Looks like there are more people with the same issue #149

**Comment by vressel** [comment]
Bericht van de helpdesk :

Beste H. De Vries, 
 
Onze excuses voor het ongemak. 
 
Helaas iets minder goed nieuws wat betreft de live productie. Deze is per heden niet meer beschikbaar sinds we overgestapt zijn naar de nieuwe beheerder. We hebben de keuze gemaakt o.b.v. de afgelopen algemene ledenvergadering om te stoppen met de live productie. 
 
Het historische verbruik blijft wel gewoon zichtbaar voor alle klanten natuurlijk.
 
Onze excuses voor het ongemak.
 
Ik wens u verder een hele prettige dag toe! 
 
Met vriendelijke groeten/Kind regards,

Sanne

Klantenservice medewerker

**Reply by jobvk** [reply]
This is some sad news, but how hard is it for the company to communicate this with her customers?

**Reply by Xorfor** [reply]
I got the same e-mail today

**Comment by Xorfor** [comment]
The e-mail I got (slightly different):
Bedankt voor uw bericht.
 
Het klopt inderdaad dat de live productie niet meer werkt. We zijn recent overgestapt naar een nieuwe beheerder. Hiervoor hebben we de keuze gemaakt o.b.v. de laatste algemene ledenvergadering (zie notules die binnenkort komen) 

Het historische verbruik blijft wel gewoon zichtbaar voor onze klanten. Er zal binnenkort een algemeen bericht verschijnen in Mijn Windcentrale hierover. 
 
Hopelijk geeft dit voor nu wat meer duidelijkheid.
 
Ik wens u verder een hele prettige dag toe! 
 
Met vriendelijke groeten/Kind regards,

Sanne

Klantenservice medewerker

**Comment by Simanias** [comment]
Ik heb de volgende e-mail verstuurd en adviseer iedereen dit ook te doen. Er is namelijk geen agenda punt opgebracht in de uitnodiging voor de ledenvergadering met betrekking tot het stopzetten van deze service.

> Geachte heer/mevrouw,
> 
> Tot mijn verbazing verneem ik via andere kanalen dat windcentrale is gestopt met de live productie informatie met betrekking tot de windmolens. Dit zou zijn besloten tijdens de laatste jaarvergadering.
> Echter ik heb de agenda doorgenomen en hierin wordt niet gesproken over een agendapunt met betrekking het stopzetten van deze service.
> Het kan dan ook niet zo zijn dat dit eenzijdig wordt doorgevoerd, zonder dat de leden op de juiste manier worden geraadpleegd. Mocht ik namelijk hiervan op juiste manier op hoogte zijn gesteld, middels de correcte agenda, dan had ik bezwaar aangetekend.
> Ik verzoek u dan ook zo spoedig mogelijk met een oplossing te komen waarbij u op de correcte manier de leden raadpleegt mocht u deze maatregel alsnog willen doorvoeren.

**Comment by ghost** [comment]
I have also sent a message to the windcentrale. In the same message also made a comment about the lake of communication.

But to be fair. In my invitation there is a point named 'Nieuwe technische asset manager'. Probably that's where it would have been discussed.

**Comment by vressel** [comment]
It seems the live-data is not coming back. Because I love the energy-dashboard, would it be possible to add a sensor that daily adds the value of my shares instead of sensor.name_energy_shares?

**Reply by jobvk** [reply]
https://github.com/jobvk/Home-Assistant-Windcentrale/discussions/150
